pages/common/git-clear-soft.md

```diff
@@ -0,0 +1,9 @@
+# git clear-soft
+
+> Clear a Git working directory as if it was freshly cloned with the current branch excluding files in `.gitignore`.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-clear-soft>.
+
+- Reset all tracked files and delete all untracked files:
```

Comment:
We should mention how this is different from #6614.

Reply:
I removed the last bit of wording since it is a standard for only files not in .gitignore to be edited and reset.......

Reply:
I don't see any changes here though @CleanMachine1?

Reply:
What do you mean?

Reply:
You said:

> I removed the last bit of wording....


....yet I see no  commits on this PR, @CleanMachine1?

Reply:
~Its in the original commit for this PR (https://github.com/tldr-pages/tldr/pull/6616/commits/085866bc6a01640d5230272efb7a3339c15f14c7)~

~Just to clarify, all that is changed from Line 7 in this PR from the PR mentioned is that I removed the explicit warning about clearing everything~

We could just add in the description of this command 

'See also: `git clear`'

I hope this is what you mean, if not just change/commit anything you see correct

Reply:
@sbrl the difference to `git-clear` is already in the command description: `excluding files in .gitignore.`

Reply:
I think @ sbrl was referring to that there is another command very similar